hlxsites/maidenform
# Initial version of minicart

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Part of #88 

Limitations:
- Guest cart is not working
- Edit functionality is missing

Steps for testing:
- Create a customer account at https://master-7rqtwti-c5v7sxvquxwl4.eu-4.magentosite.cloud
- Go to https://develop--helix-venia-jam--magento-comops.hlx.page/ and sign in
- Add something to your cart
- Via developer console, copy the value of the `STOREFRONT_SDK-STORE` key in local storage and paste it to https://issue-88-mfe--maidenform--hlxsites.hlx.page/
- Reload the Franklin page and observe the same cart content

Test URLs:
- Before: https://main--maidenform--hlxsites.hlx.page/
- After: https://issue-88-mfe--maidenform--hlxsites.hlx.page/
